README.md

```diff
@@ -65,89 +121,176 @@ label_maker = cp.LabelMaker(
     window_size="1h",
 )
 ```
+
 ### Search Labels
-Next, we automatically search and extract the labels by using [`LabelMaker.search`](http://docs.compose.ml/en/latest/generated/methods/composeml.LabelMaker.search.html#composeml.LabelMaker.search).
+Next, we automatically search and extract the labels by using [`LabelMaker.search`](https://compose.featurelabs.com/en/latest/generated/methods/composeml.LabelMaker.search.html#composeml.LabelMaker.search). The data must be sorted by the time index before passing into the label maker. For more details on how the label maker works, see [Main Concepts](https://compose.featurelabs.com/en/latest/main_concepts.html).
```

Comment:
do we need the line "The data must be sorted by the time index before passing into the label maker."? I think we can just raise an informative warning when that is the case. the user doesn't need to learn it from the getting started

Reply:
Okay, will remove line.